synapse/tools/cryo/cat.py

```diff
@@ -17,6 +21,7 @@ def main(argv, outp=s_output.stdout):
     pars.add_argument('--size', default=10, type=int, help='How many items to display')
     pars.add_argument('--timeout', default=10, type=int, help='The network timeout setting')
     pars.add_argument('--authfile', help='Path to your auth file for the remote cell')
+    pars.add_argument('--jsonl', default=False, action='store_true', help='Output items in jsonl format')
```

Comment:
I'd throw a ``--verbose`` option here for enabling / disabling debug logging

Reply:
ok